### PR TITLE
fix detection of existing fallback

### PR DESCRIPTION
**Pull request by TrySound**

### CODE REVIEW

**Comment by TrySound** [comment]
@MoOx @magsout  Is it expected behaviour?

File: index.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ module.exports = function plugin() {
       // if previous prop equals current prop
       // if previous prop has hexadecimal value and current prop has rgba() value
       // no need fallback
-      if (decl.prev() && decl.prev().prop === decl.prop && decl.prop.indexOf("rgba") === decl.prev().prop.indexOf("#")) {
+      if (decl.prev() && decl.prev().prop === decl.prop && decl.value.indexOf("rgba") === decl.prev().value.indexOf("#")) {
```
Comment (magsout) [inline index.js]: oh yes, you're right. test always returned true due to -1 result ..

File: test/fixtures/rgba-double-fallback.css [hunk 2]
```diff
@@ -0,0 +1,7 @@
+.foo {
+  background: rgba(153, 221, 153, 0.8);
+  background: rgba(153, 221, 153, 0.8);
```
Comment (magsout) [inline test/fixtures/rgba-double-fallback.css]: Perhaps we should remove this line in this case. `decl.prop === dec.prev.().prop &&  decl.value === decl.prev(). value`

But I do not know if this is the plugin role

**Comment by magsout** [comment]
Please when you do a PR, if you can increase the version in package.json and add a note in the readme

File: test/fixtures/rgba-double-fallback.css (hunk 2 above)
Comment (TrySound) [inline test/fixtures/rgba-double-fallback.css]: This is a task of minifiers.

**Comment by TrySound** [comment]
@magsout squash commits? Or leave two?

**Comment by magsout** [comment]
squash plz

File: test/fixtures/rgba-double-fallback.css (hunk 2 above)
Comment (magsout) [inline test/fixtures/rgba-double-fallback.css]: yes.

**Comment by TrySound** [comment]
patch or minor?

**Pull request merged**